### PR TITLE
Updated to Ember/Ember-CLI 2.6.0 added ember-prop-types

### PR DESCRIPTION
**Pull request by juwara0**
# MINOR
# CHANGELOG
- **Added** `ember-prop-types`.
- Upgraded to 2.6.0 Ember, Ember CLI, and Ember Data.
- Upgrade dependencies.

### CODE REVIEW

File: tests/dummy/app/pods/demo/controller.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import Ember from 'ember'
 
-const {Controller} = Ember
+const {A, Controller} = Ember
 
 export default Controller.extend({
 
-  tableSortList: [
+  tableSortList: A([
```
Comment (rox163) [inline tests/dummy/app/pods/demo/controller.js]: would this affect consuming apps? Do they have to change any sort lists they use into Ember Arrays?

File: tests/helpers/module-for-acceptance.js [hunk 2]
```diff
@@ -1,23 +1,23 @@
 import { module } from 'qunit'
```
Comment (rox163) [inline tests/helpers/module-for-acceptance.js]: I think this file is qunit specific? If so we can delete it.

File: bower.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "ember-frost-sort",
   "dependencies": {
-    "ember": "~2.3.0",
-    "ember-cli-shims": "0.1.0",
+    "ember": "~2.6.0",
```
Comment (rox163) [inline bower.json:4]: You need to update the resolutions as well on line 18

File: package.json [hunk 4]
```diff
@@ -47,7 +47,6 @@
     "ember-computed-decorators": "^0.2.2",
     "ember-concurrency": "^0.7.8",
     "ember-data": "~2.6.0",
-    "ember-disable-prototype-extensions": "^1.1.0",
```
Comment (rox163) [inline package.json]: aha! nice.

**Comment by rox163** [comment]
:+1:
with the bower fix it should pass the build.

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-sort/pull-request/20/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-sort)

File: tests/helpers/module-for-acceptance.js (hunk 2 above)
Comment (juwara0) [inline tests/helpers/module-for-acceptance.js]: Done.

File: bower.json (hunk 3 above)
Comment (juwara0) [inline bower.json:4]: Done.

**Comment by rox163** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-sort/pull-request/20/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-sort)

**Pull request merged**